processing/src/test/java/io/druid/query/topn/AlphaNumericTopNMetricSpecTest.java

```diff
@@ -104,8 +104,8 @@ public void testSerdeAlphaNumericTopNMetricSpec() throws IOException
                        + "    \"previousStop\": \"test\"\n"
                        + "}";
     ObjectMapper jsonMapper = TestHelper.makeJsonMapper();
-    TopNMetricSpec actualMetricSpec = jsonMapper.readValue(jsonMapper.writeValueAsString(jsonMapper.readValue(jsonSpec, TopNMetricSpec.class)), AlphaNumericTopNMetricSpec.class);
-    TopNMetricSpec actualMetricSpec1 = jsonMapper.readValue(jsonMapper.writeValueAsString(jsonMapper.readValue(jsonSpec1, TopNMetricSpec.class)), AlphaNumericTopNMetricSpec.class);
+    TopNMetricSpec actualMetricSpec = jsonMapper.readValue(jsonMapper.writeValueAsString(jsonMapper.readValue(jsonSpec, TopNMetricSpec.class)), TopNMetricSpec.class);
```

Comment:
are all these test changes meant to be in this PR?

Reply:
Yes, it causes a syntax error due to the changed Jackson version.